### DISCUSSION TITLE
Welcome to CryptoEconLab Discussions!

### DISCUSSION THREAD

**Discussion by KarolaKirsanow** [protocol/CryptoEconLab#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JithinKS97** [comment]
Hello community, my name is Jithin. I'm a software developer from India, currently working in a startup building, analytics tools for creators. I think crypto-economics is a discipline that has the potential to coordinate humans together to build a better world. I'm here to learn more about the area and if possible contribute in whatever ways that I can. What is the best place to start learning so that I can be a part of what you guys are doing?

**Reply by KarolaKirsanow** [reply]
Hi @JithinKS97 ! Your post has inspired us to assemble some of our favorite resources on cryptoeconomic topics to share with the community - stay tuned! In the meantime, you can check out the references cited in the [Open Problems](https://github.com/protocol/CryptoEconLab/tree/main/Open_Problems) to get an idea of the various fields having intersections with CEL research.

**Comment by gobengo** [comment]
Hey I’m @gobengo and what I am trying to do is computational cooperativism
